tools/runner/update_manifest.py

```diff
@@ -7,21 +7,31 @@
 
 root = localpaths.repo_root
 
-from manifest import manifest, update
+from manifest import manifest
 
 def main(request, response):
     path = os.path.join(root, "MANIFEST.json")
 
-    manifest_file = None
-    try:
-        manifest_file = manifest.load(root, path)
-    except manifest.ManifestVersionMismatch:
-        pass
-    if manifest_file is None:
-        manifest_file = manifest.Manifest("/")
-
-    update.update(root, manifest_file)
-
-    manifest.write(manifest_file, path)
-
-    return [("Content-Type", "application/json")], json.dumps({"url": "/MANIFEST.json"})
+    # TODO make this download when required
```

Comment:
Is this implying that:

- this always downloads, even when it shouldn't, or
- this never downloads, but should sometimes

(I'm not familiar with what 'downloads' means here)

Reply:
"This never downloads, but should, sometimes". I updated the comment a little.